webpages/settings/components/category-selector.html

```diff
@@ -59,17 +61,19 @@
     opacity: 0;
   }
 
-  .category:hover,
-  .category.sel {
+  .category:hover {
     background: var(--hover-darken);
   }
+  .category.sel {
+    background: var(--sel-darken);
+  }
   .category::before {
     content: "";
     transition: all 0.2s ease;
     display: block;
     width: 4px;
-    border-radius: 0 4px 4px 0;
-    height: 40px;
+    border-radius: 4px 0 0 4px;
```

Comment:
The RTL `border-radius` will also need to be reversed.

Reply:
oh ye

Reply:
Line 84 should be changed to `0 4px 4px 0`, I just couldn't add a review on the unchanged line.

Reply:
> Line 84 should be changed to `0 4px 4px 0`, I just couldn't add a review on the unchanged line.

resolved, sorry for a long time